godot/src/ui/components/button_touch_action/button_touch_action.gd

```diff
@@ -1,22 +1,48 @@
 @tool
-extends TextureButton
+extends Control
 
 @export var text_color = Color.WHITE:
 	set(new_value):
-		$Label.add_theme_color_override("font_color", new_value)
+		%Label_Letter.add_theme_color_override("font_color", new_value)
```

Comment:
Use `@onready label_letter = %Label_Letter`

Reply:
> Use `@onready label_letter = %Label_Letter`

I can't, because this script has the @tool on the top. So the _ready is processed after the sets